quickstarts/microsoft.operationsmanagement/azure-sentinel/main.bicep

```diff
@@ -1,27 +1,33 @@
-param workspaceName string
 param location string = resourceGroup().location
+param sentinelName string = 'sentinel'
+param sku string = 'PerGB2018'
+param retentionInDays int = 90
```

Comment:
is there a min and mix for this?

Reply:
Indeed, I can set min and max for retentionInDays parameter